### DISCUSSION TITLE
몸이 안좋아 이번 주차 참석이 힘들것 같습니다

### DISCUSSION THREAD

**Discussion by wooyaggo86** [ThinkAboutSoftware/AcademicConference#188] [General]
부스터 샷을 맞고나서 컨디션이 안좋아 쉬고 있습니다.
다들 건강 조심하시고 즐거운 시간 보내세요🙏

**Comment by jongfeel** [comment]
알겠습니다. 건강이 제일 중요하죠.
잘 쉬시고 다음 모임은 마지막 책 시작이니까 함께 했으면 좋겠습니다.
